editor/patches/react-select+3.1.0.patch

```diff
@@ -1,7 +1,6 @@
 diff --git a/node_modules/react-select/src/.DS_Store b/node_modules/react-select/src/.DS_Store
 new file mode 100644
-index 0000000..dbd07c1
-Binary files /dev/null and b/node_modules/react-select/src/.DS_Store differ
```

Comment:
what's up with this line? is this noise?

Reply:
ahh the DS_Store is something that must have been added by macOS!

Reply:
but that is getting deleted, so… i think we're good here. prob shouldn't have been there in the first place.